### ISSUE TITLE
Please port latest version to sonification for interview

### ISSUE THREAD

**Issue by terracoda** [phetsims/faradays-law#137]
We need an up to date described sonified version by end of @zepumph's day today. There is an early interview (9:30 am ET) Tuesday morning.

If at all possible please address the following issues before the port:
- [x]  #136 - No accessible name for Bar Magnet.

If the above cannot be fixed, I can just make a note about the issue for the interviewers.

@zepumph please coordinate with @mbarlow12 regarding issue #136.

**Comment by terracoda** [comment]
@zepumph, I am not sure if these requests warrant the dev:sonifcation label.

**Comment by terracoda** [comment]
@zepumph, as for links i will need:

- new dev link for described version if changed from 
https://phet-dev.colorado.edu/html/faradays-law/1.4.0-dev.46/phet/faradays-law_en_phet.html?accessibility

- dev link for latest described sonified version

**Comment by zepumph** [comment]
@mbarlow12 I think the easiest way to proceed is to in the following:

* @mbarlow12 to complete as much as possible in the simulation for the upcoming interview today.
* @mbarlow12 to create a dev deploy with `--brands=phet,phet-io` so that we can also get the phet-io version for the sonification wrapper
* @zepumph to add that dev version to the sonification wrapper and deploy that.

If the ball isn't in my court by 3pm MT today, then I will just do the necessary actions with what is currently in master.

**Comment by terracoda** [comment]
@mbarlow12 and @zepumph - interview is 9:30 am ET Tuesday (i.e., Sept 25) and not today.

**Comment by mbarlow12** [comment]
@zepumph @terracoda New html & wrapper versions deployed:
- https://phet-dev.colorado.edu/html/faradays-law/1.4.0-dev.53/phet/faradays-law_en_phet.html
- https://phet-dev.colorado.edu/html/faradays-law/1.4.0-dev.53/phet-io/

I'll keep myself assigned until this is closed.

**Comment by zepumph** [comment]
Great thanks!

**Comment by zepumph** [comment]
@mbarlow12 thank you for doing this. 

Here is the new sonification link:

https://phet-dev.colorado.edu/html/phet-io-wrapper-sonification/1.0.0-dev.77/phet-io-wrapper-sonification/sonification.html

@mbarlow12 from now on, to support the sonification wrapper as it applies to faradays-law, please use the above commit as a guide to update the FL version. Then:
* commit and push
* `cd phet-io-wrapper-sonification`
* `grunt wrapper` to deploy the simulation

**Comment by terracoda** [comment]
@mbarlow12 and @zepumph, thanks to both of you for getting these links out so quickly!

**Comment by terracoda** [comment]
I'll close this issue now as interview(s) took place.
